src/stats.php

```diff
@@ -116,9 +116,10 @@ function executeContributionGraphRequests(string $user, array $years): array
  * Get all HTTP request responses for user's contributions
  *
  * @param string $user GitHub username to get graphs for
+ * @param int|null $startingYear Override the minimum year to get graphs for
  * @return array<stdClass> List of contribution graph response objects
  */
-function getContributionGraphs(string $user): array
+function getContributionGraphs(string $user, ?int $startingYear=null): array
```

Comment:
Minor change to formatting
```suggestion
function getContributionGraphs(string $user, ?int $startingYear = null): array
```

Reply:
Done @DenverCoder1 !